Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN apt-get update \
 FROM main as bsbuild
 ENV TZ=UTC
 ENV DEBIAN_FRONTEND=noninteractive
-ADD https://bluespice.com/filebase/bluespice-free-4-2/ /opt/BlueSpice-free.zip
+ADD https://buildservice.bluespice.com/dpackages/BlueSpice-free-4.2.2.tar.gz /opt/BlueSpice-free.tar.gz
```

Comment:
Apparently this is only accessible from certain IP addresses. This is totally acceptable for me, but I wanted to mention.

Reply:
so maybe if we can have a public package serving that would be nice...